docs/source/getting_started/index.rst

```diff
@@ -90,58 +78,44 @@ In order to get those files, you need to clone the
   save some time and bandwidth. If you want to get a full commit history, you
   can omit this option.
 
+We should adjust Iroha config to match Postgres host it will connect to. Edit ``host`` parameter of ``config.sample`` file in ``./iroha/example`` directory. Set it to ``some-postgres``.
```

Comment:
We can eliminate this extra step if `docker/release/entrypoint.sh` will use `config.docker` instead of `config.sample`. 

I suggest to modify `entrypoint.sh` and left here a note about `iroha:latest` till master branch would not be updated.

Reply:
To be honest, when I was writing this dock NOT using an entrypoint.sh and entering `irohad` command was made on purpose, so newcomers have a chance to understand what's going on in the container (no autostarting everything)

Reply:
I will open a PR to use `config.docker` instead of 'config.sample'